stack.yaml

```diff
@@ -2,13 +2,13 @@
 # For more information, see: http://docs.haskellstack.org/en/stable/yaml_configuration.html
 
 # Specifies the GHC version and set of packages available (e.g., lts-3.5, nightly-2015-09-21, ghc-7.10.2)
-resolver: lts-5.1
+resolver: lts-5.9
 
 # Local packages, usually specified by relative directory name
 packages:
 - '.'
 # Packages to be pulled from upstream that are not in the resolver (e.g., acme-missiles-0.3)
-extra-deps: []
+extra-deps: [aeson-0.11.0.0]
```

Comment:
no it's actually 0.11.0.0, and as far as i know there's no way of saying `>=0.11` in extra deps.
But i know that one is working for now and higher versions like 0.11.1.3 gave me some dependency conflict.